### ISSUE TITLE
CocoaDebug integration with OSLog/os_log

### ISSUE THREAD

**Issue by PushpakN** [CocoaDebug/CocoaDebug#92]
Is there a work around where CocoaDebug can display logs added by os_log other than writing a wrapper and calling print() and NSLog() from it.

**Comment by CocoaDebug** [comment]
NSLog() does not need a wrapper. Just print() need a wrapper.

**Comment by PushpakN** [comment]
@CocoaDebug Yes you are right. Wanted to know is there a way to get CocoaDebug with os_log.
